api/app/tests/Unit/Repository/CourtOrderRepositoryTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
```

Comment:
Move to the integration directory and it should be fine. The namespace gives a clue as to where it should sit.